.github/workflows/continuous-integration-cypress.yml

```diff
@@ -32,5 +36,5 @@ jobs:
       - uses: actions/upload-artifact@v1
         if: failure()
         with:
-         name: screenshots
+         name: screenshots/${{ github.event.workflow_run.environment }}
```

Comment:
As this job now runs against different environments, I added a suffix to distinguish. Not sure if you require this or not @jordan-beard